xrpl/models/base_model.py

```diff
@@ -15,6 +15,24 @@
 from xrpl.models.required import REQUIRED
 from xrpl.models.types import XRPL_VALUE_TYPE
 
+
+def is_snake_case(k: str) -> bool:
```

Comment:
There isn't really a need for this helper function, since it's just one line and that line is pretty self explanatory.